packages/core/src/utils/variant-group.ts

```diff
@@ -17,18 +17,29 @@ interface VariantGroup {
   items: HighlightAnnotation[]
 }
 
-export function parseVariantGroup(str: string | MagicString, separators = ['-', ':'], depth = 5) {
+export function parseVariantGroup(str: string | MagicString, separators = ['-', ':'], depth = 5, type?: string) {
```

Comment:
Where the `type` used for, other than checking if it presents? Should be make a literal enum like `'shourtcut' | 'rule' | ...`?

Reply:
I changed it to a boolean type. It seems that I only need to determine whether I need to enable the conversion of attributes. I changed it to a boolean type. It seems that I only need to determine whether I need to enable the conversion of attributes.